docs/topics/native/native-spm.md

```diff
@@ -132,7 +164,7 @@ To set up the publishing of an XCFramework:
     
     The output will describe any errors found or show the successful download and parsing result if the manifest is correct.
 
-6. Push the `Package.swift` file to your remote repository. Make sure to create and push a Git tag with the
+7. Push the `Package.swift` file to your remote repository. Make sure to create and push a Git tag with the
    semantic version of the package.
 
 ### Add the package dependency
```

Comment:
Let's mention that now we can add the package dependency to a client iOS project.

Reply:
Also this project appears suddenly, so possibly we can mention that you can use your own client iOS project or create a new one. It makes sense because we mention ContentView.swift file in "Check your setup section", but this step valid only for new created project.